tests/test_slack_integration.py

```
@@ -0,0 +1,79 @@
+from robusta.core.reporting.base import Finding, FindingSeverity, FindingSource
+from robusta.core.sinks.slack.slack_sink_params import SlackSinkParams, SlackTemplateStyle
+from robusta.core.reporting.blocks import MarkdownBlock
+from robusta.integrations.slack.sender import SlackSender
+from tests.config import CONFIG
+from tests.utils.slack_utils import SlackChannel
+
+TEST_ACCOUNT = "test account"
+TEST_CLUSTER = "test cluster"
+TEST_KEY = "test key"
+
+@pytest.mark.parametrize("template_style,custom_template,expected_phrases", [
```

Ruff (0.8.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)